content/docs/reference/config/installation-options/index.md

```diff
@@ -0,0 +1,250 @@
+| `prometheus.service` |  |  |
+| `prometheus.service.annotations` | `{}` |  |
+| `prometheus.service.nodePort` |  |  |
+| `prometheus.service.nodePort.enabled` | `false` |  |
+| `prometheus.service.nodePort.port` | `32090` |  |
+| `servicegraph` | `servicegraph.local` |  |
+| `servicegraph.enabled` | `false` |  |
+| `servicegraph.replicaCount` | `1` |  |
+| `servicegraph.image` | `servicegraph` |  |
+| `servicegraph.service` |  |  |
+| `servicegraph.service.annotations` | `{}` |  |
+| `servicegraph.service.name` | `http` |  |
+| `servicegraph.service.type` | `ClusterIP` |  |
+| `servicegraph.service.externalPort` | `8088` |  |
+| `servicegraph.service.internalPort` | `8088` |  |
+| `servicegraph.ingress` | `servicegraph.local` |  |
+| `servicegraph.ingress.enabled` | `false` |  |
+| `servicegraph.ingress.hosts` | `servicegraph.local` |  |
+| `servicegraph.ingress.annotations` |  |  |
+| `servicegraph.ingress.tls` |  |  |
+| `servicegraph.prometheusAddr` | `http://prometheus:9090` |  |
+| `zipkin` |  |  |
+| `zipkin.enabled` | `true` |  |
+| `tracing` | `zipkin.local` |  |
+| `tracing.enabled` | `false` |  |
+| `tracing.jaeger` |  |  |
+| `tracing.jaeger.enabled` | `false` |  |
+| `tracing.jaeger.memory` |  |  |
+| `tracing.jaeger.memory.max_traces` | `50000` |  |
+| `tracing.replicaCount` | `1` |  |
+| `tracing.image` |  |  |
+| `tracing.image.repository` | `jaegertracing/all-in-one` |  |
+| `tracing.image.tag` | `1.5` |  |
+| `tracing.service` |  |  |
+| `tracing.service.annotations` | `{}` |  |
+| `tracing.service.name` | `http` |  |
+| `tracing.service.type` | `ClusterIP` |  |
+| `tracing.service.externalPort` | `9411` |  |
+| `tracing.service.internalPort` | `9411` |  |
+| `tracing.service.uiPort` | `16686` |  |
+| `tracing.ingress` | `zipkin.local` |  |
+| `tracing.ingress.enabled` | `false` |  |
+| `tracing.ingress.hosts` | `zipkin.local` |  |
+| `tracing.ingress.annotations` |  |  |
+| `tracing.ingress.tls` |  |  |
+| `kiali` |  |  |
+| `kiali.enabled` | `false` |  |
+| `kiali.replicaCount` | `1` |  |
+| `kiali.image` |  |  |
+| `kiali.image.repository` | `kiali/kiali` |  |
+| `kiali.image.tag` | `v0.4.0` |  |
+| `kiali.ingress` |  |  |
+| `kiali.ingress.enabled` | `false` |  |
+| `kiali.ingress.annotations` |  |  |
+| `kiali.ingress.tls` |  |  |
+| `kiali.dashboard` |  |  |
+| `kiali.dashboard.username` | `admin` |  |
+| `kiali.dashboard.password` | `admin` |  |
+<!-- AUTO-GENERATED-END -->
+
+The Helm chart also offers significant customization options per individual
+service. Customize these per-service options at your own risk. The per-service options are exposed via
+the [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) file.
+
+## Example: traffic management minimal set
+
+Istio has a rich feature set, but you may only want to use a subset of these. For instance, you might be only interested in installing the minimum necessary to support traffic management functionality.
+
```

Comment:
This is burying the lead :). If this is meant to serve as an example of a la carte Istio, this should go into main docs, as its own thing: essentially, something like users wishing to play with Istio's networking features can start with a minimal Istio install consisting of Pilot and an Ingress Gateway, as shown below:..

Reply:
Should we move this example back to the helm setup page?

Reply:
yes please move this back to the main page.